code/modules/cargo/packs/costumes_toys.dm

```diff
@@ -293,3 +293,18 @@
 	for(var/i in 1 to 10)
 		cardpacktype = pick(subtypesof(/obj/item/cardpack))
 		new cardpacktype(C)
+
+/datum/supply_pack/costumes_toys/stickers
+	name = "Sticker Crate"
+	desc = "This crate contains a random assortment of stickers."
+	cost = CARGO_CRATE_VALUE * 3
+	contains = list()
+
+/datum/supply_pack/costumes_toys/stickers/fill(obj/structure/closet/crate/C)
```

Comment:
Not a requirement but maybe these could go in a box in the crate maybe? Feels weird to open a crate to find 4 loose stickers, but it's probably fine.